src/features/scheduling/BunkJamboreeScheduler.ts

```
@@ -0,0 +1,31 @@
+
+export class BunkJamboreeScheduler {
+  schedule: SectionSchedule<"BUNK-JAMBO"> = { blocks: {}, alternatePeriodsOff: {} };
+
+  bunks: Bunk[] = [];
+  admins: AdminAttendeeID[] = [];
+
+  preferences: Preferences = {};
+
+  blocksToAssign: string[] = [];
+
+  constructor() { return this; }
```

Biome (2.1.2)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor may confuse users of the class.

(lint/correctness/noConstructorReturn)